image/tools/lib/component/enmasse_pv.sh

```diff
@@ -32,18 +33,30 @@ function component_dump_data {
         exit 0
     fi
 
-    mkdir -p ${dump_dest}
+    mkdir -p ${archive_temp}
 
     for pod in ${pods}; do
         timestamp_echo "Processing enmasse broker pod ${pod}"
+        mkdir -p ${dump_dest}
         dump_pod_data ${pod} ${dump_dest}
+        ls ${dump_dest}/*
+        if [ "$?" -eq "0" ]; then
+            local ts=$(date '+%H_%M_%S')
+            timestamp_echo "Creating archive for pod ${pod}"
+            tar -zcvf "$archive_temp/${pod}-${ts}.tar.gz" -C $dump_dest .
+            rm -rf $dump_dest
+        else
+            timestamp_echo "No enmasse broker data to backup for pod: ${pod}"
+    fi
```

Comment:
The problem with this approach is that the structure of the end-resulting archive is different: it contains other archives. I think this would be considered a breaking change, since any restoration SOPs (we may not know about all of these) would no longer be correct.